AmplifyTestCommon/Helpers/AuthHelper.swift

```diff
@@ -1,107 +0,0 @@
-//
-// Copyright Amazon.com Inc. or its affiliates.
-// All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-
-import Foundation
-import AWSMobileClient
-
-class AuthHelper {
```

Comment:
are all the plugin tests off of this AuthHelper?

Reply:
Yes, couldnot find any reference to this one.